### ISSUE TITLE
Feedback: PF2 CSS compatibility with Dorako UI 

### ISSUE THREAD

**Issue by apoapostolov** [Larkinabout/fvtt-token-action-hud-core#53]
**I will start with a disclaimer this is not a negative feedback for the new PF2 skin, it is wonderful if a bit "thick" because of the extra borders on each item. but looks very stylish.** 

However, the new PF2 skin is very much in contrast with the most popular, semi-official CSS skin mod for PF2 called Dorako UI. I feel that a direct collaboration with Dorako UI is a better to approach that will work for more people. Since Dorako puts a lot of effort into making a dark-mode UX for PF2, it could allow your skin to work with his dark-mode system-wide variables. 

Again, just trying to raise awareness of Dorako's work, not a hostile take on your work. Feel free to close this issue, or we could discuss it if you have an opinion. Thanks in advance.

**Comment by Larkinabout** [comment]
I think it's just about choice. The Dorako UI theme has been supported by the original Token Action HUD for quite a while and I ported over the style for this reason. The new style isn't an attempt to compete with, or replace, it. I would like to expand on the available styles for all users regardless of system, so they can have a Pathfinder themed HUD while playing D&D or a D&D themed HUD while playing Pathfinder if they want.

I called the style Pathfinder because it's based off the style of the core Pathfinder 2 character sheet. The default even when installing the PF2 version of the new Token Action HUD will still be the Foundry VTT style. I expect users who love the Dorako UI theming will choose the matching HUD style.
